client/src/components/LogInModal.jsx

```diff
@@ -13,26 +13,22 @@ const LogInModal = ({ isOpen, onClose }) => {
     //Call useNavigate hook to change component display upon log in
     const navigate = useNavigate();
 
+    //State to hold various error messages
+    const [formErrorMessage, setFormErrorMessage] = useState(null);
+
     const handleSubmit = async (event) => {
         event.preventDefault();
-        
-        //Check that correct value is being submitted right now
-        console.log(event.currentTarget.username.value);
-
-        //Set the value of the username input field to be stored in username variable
-        const username = event.currentTarget.username.value;
+        setFormErrorMessage(null);
 
-        //Future Plans: Add in password functionality - OAuth?
-
-        //If on submission, there is a username submitted, send it to the server to check its existence
-        if (username) {
+        //If on submission, both credentials are submitted, send them to the server to check authorization
+        if (credentials.email && credentials.password) {
```

Comment:
Another error check is to verify it's not just spaces entered.  React may already accommodate for that but you can easily test if that's the case.

Reply:
Ooh great idea, thank you! I've added that check with commit [0ad41c2](https://github.com/tati-guzman/click-and-treat/pull/8/commits/0ad41c2b03dc582b792f45aa55d861e33db745dc)